File: 26. Remove Duplicates from Sorted Array/Solution.py
class Solution:
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums) < 2:
            return len(nums)
        i, j = 0, 1
        while j < len(nums):
            if nums[i] < nums[j]:
                i += 1
                nums[i] = nums[j]
            j += 1
        return i + 1

File: 26. Remove Duplicates from Sorted Array/Solution.cpp
class Solution
{
  public:
    int removeDuplicates(vector<int> &nums)
    {
        if (nums.size() < 2)
            return nums.size();
        int i, j;
        for (i = 0, j = i + 1; j < nums.size(); j++)
        {
            if (nums[i] < nums[j])
            {
                nums[++i] = nums[j];
            }
        }
        return i + 1;
    }
};